python/py-fabio/Portfile

```diff
@@ -23,13 +23,16 @@ long_description \
 
 homepage            https://github.com/silx-kit/fabio
 
-checksums           rmd160  9e3b2cad4d3883eb3c71a05553f186506b31175e \
-                    sha256  04dbb312b42462dfe09685ce9ff1d7494639eef3f2a32e2ed0bb8ee284f865bc \
-                    size    1511482
+checksums           rmd160  d8493e98494abc48b5d4b75ede069a49db9f9e81 \
+                    sha256  42d2b04cc602a018bda1a6822725a75148674ab561410a0f99036dd9361c624a \
+                    size    1522119
 
-python.versions     37 38 39
+python.versions     37 38 39 310
 
 if {${name} ne ${subport}} {
+    depends_build-append \
```

Comment:
I doubt this is necessary since upstream doesn't mention it (see comment in other PR)

Reply:
See #15010 conversation about `py-pythran` as a build dependency. If I remove that dependency from the Portfile, then I find I cannot install this port unless I activate pythran AND install not-in-trace mode.

That is, these 3 commands fail:
```
sudo port deactivate py310-pythran && sudo port -vs install py310-fabio
sudo port deactivate py310-pythran && sudo port -vst install py310-fabio
sudo port activate py310-pythran && sudo port -vst install py310-fabio
```
but this one succeeds:
```
sudo port activate py310-pythran && sudo port -vs install py310-fabio
```

Reply:
yeah, well that's an issue with trace-mode but this port does not need `pythran` to build so please remove it. The buildbots will not have it installed when building this port and it works fine when `pythran` is not present.

I will remove if from the port for which I already merged the PR.

Reply:
uninstall the `pythran` port, clean the fabio port and install again - it will work

Reply:
Okay. Right. Clearly the `port clean` has some effect that I didn't expect. I'll update this PR and #15010 too.

Reply:
> I will remove if from the port for which I already merged the PR.

I think you're referring to #15009. I was just noticing that it only supports EOL versions of Perl (5.28 and 5.30). I am happy to revise it to use Perl 5.32 or 5.34 only. I could remove the build-dep on pythran at the same time, if you like.

Reply:
NVM. I see you did it already.